tests/market.test.ts

```diff
@@ -854,40 +854,44 @@ describe("Market", () => {
 				odds,
 				oracleSigner
 			);
+
 			const oracleOwner = await oracle.getOwner();
 			expect(oracleOwner).to.equal(oracleSigner.address);
-			await oracle.setScratchedResult(
-				formatBytes16String(marketId),
-				formatBytes16String(propositionId),
-				odds,
-				signature
-			);
+			expect(
+				await oracle.setScratchedResult(
+					formatBytes16String(marketId),
+					formatBytes16String(propositionId),
+					odds,
+					signature
+				)
+			).to.emit(oracle, "ScratchedSet");
+
 			const index = 0;
 
-			// Scratched bets do not need to wait for the race to finsih
+			// Scratched bets do not need to wait for the race to finish
 			expect(await market.settle(index), "Issue with settling scratched bet")
 				.to.emit(market, "Settled")
 				.withArgs(index, betPayout, SCRATCHED, bob.address);
 
-			const newNftBalance = await market.balanceOf(bob.address);
-			expect(newNftBalance).to.equal(0, "Bob should have no NFTs now");
-
-			await expect(market.settle(index)).to.be.revertedWith(
-				"settle: Bet has already settled"
-			);
-			exposure = await market.getTotalExposure();
-			expect(exposure).to.equal(0);
-
-			inPlay = await market.getTotalInPlay();
-			expect(inPlay).to.equal(0);
-
-			// Everything should be as it was before
-			const balance = await underlying.balanceOf(bob.address);
-			const endVaultBalance = await underlying.balanceOf(vault.address);
-			const endMarketBalance = await underlying.balanceOf(market.address);
-			expect(balance).to.equal(bobBalance);
-			expect(vaultBalance).to.equal(endVaultBalance);
-			expect(marketBalance).to.equal(endMarketBalance);
+			// const newNftBalance = await market.balanceOf(bob.address);
```

Comment:
Did you mean to leave this commented out?

Reply:
Nope.  Fixed.